components/brave_wallet/browser/brave_wallet_service.cc

```diff
@@ -352,8 +352,8 @@ bool BraveWalletService::AddUserAsset(mojom::BlockchainTokenPtr token,
   if (network_id.empty())
     return false;
 
-  // Verify input token ID for ERC721.
-  if (token->is_erc721) {
+  // Verify input token ID for ERC721 and ERC1155 tokens
+  if (token->is_erc721 || token->is_erc1155) {
```

Comment:
`bool check_token_id = token->is_erc721 || token->is_erc1155;` then use it here and below when calling FindAsset.